src/connection/index.ts

```diff
@@ -102,7 +102,7 @@ export const uniwalletWCV2ConnectConnection: Connection = {
   type: ConnectionType.UNISWAP_WALLET_V2,
   getIcon: () => UNIWALLET_ICON,
   shouldDisplay: () => Boolean(!getIsInjectedMobileBrowser() && !isNonIOSPhone),
-  isNew: true,
+  isNew: false,
```

Comment:
can we just delete `isNew` completely?

Reply:
probably? its not being used anywhere else so as long as I guess it won't be needed in the future it can probably be deleted

Reply:
Confirming it's only used for the badge and can be removed